### DISCUSSION TITLE
Review cards with  VoiceOver

### DISCUSSION THREAD

**Discussion by daviddumonde** [st3v3nmw/obsidian-spaced-repetition#492] [Q&A]
I recently had a traumatic brain injury which caused me to lose a part of my eyesight -- fortunately only a small amount but just enough that it makes reading text difficult. I've been relying on VoiceOver on my Mac to read text to me.

VoiceOver does read the text on the front of the cards, but only after it reads everything else on the screen behind the modal. It will read the back of the cards too, but only after it reads everything else on the screen behind it. Sometimes if there's specific text I want VoiceOver to read, I select the text first with my mouse and use a keyboard shortcut to invoke VoiceOver. The modal prevents me from selecting any text on the cards. It would be just fine for me if only I could select the text for VoiceOver..

I don't know whether this is a Spaced Repetition plug-in issue, or an Obsidian issue. I don't think I have any other obsidian plug-ins that create modals like this. I've only begun using VoiceOver recently, so it may be a user issue too.

Also I expected the plug-in to randomize the cards. Instead it displays in the same order every time. Is this a setting?

Thanks for any assistance you can offer!

**Comment by ronzulu** [comment]
Hi David, sorry to hear of your difficulties. I'm kind of on the other side of things – a problem with hands prevents me from doing much typing. So I've benefited immensely from speech recognition over 25+ years (I can't believe it's that long now).

I don't know anything about VoiceOver, but curious if you are still having trouble with it.

Regarding the "randomised the cards" question, yes there is a setting. Go to the Obsidian spaced repetition plug-in settings list. It's kind of in the middle and is called "Randomize card order during review?".

Note that currently the randomisation may not be completely as one would expect, e.g. it doesn't randomize across all decks being reviewed.
